theHarvester/__main__.py

```diff
@@ -1398,156 +1386,173 @@ async def handler(lst):
         except Exception as er:
             print(f'[!] An error occurred while saving the JSON file: {er} ')
         print('\n\n')
-    
+
     # Enhanced code block for API Endpoint scanning feature
     if args.api_scan or 'api_endpoints' in engines:
         try:
             # Define a default wordlist if none is specified
-            wordlist = args.wordlist if args.wordlist else "./wordlists/api_endpoints.txt"
-            
+            wordlist = args.wordlist if args.wordlist else './wordlists/api_endpoints.txt'
```

Comment:
you should use the data dir to get api_endpoints.txt file which is where you need to put the file is pop it in here https://github.com/laramies/theHarvester/tree/master/theHarvester/data/wordlists